soc/renesas/ra/ra8t1/power.c

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/kernel.h>
+#include <zephyr/pm/pm.h>
+#include <soc.h>
+#include <zephyr/init.h>
+#include <r_lpm.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(soc, CONFIG_SOC_LOG_LEVEL);
+
+lpm_instance_ctrl_t g_lpm_sleep_ctrl;
+const lpm_cfg_t g_lpm_sleep_cfg = {
+	.low_power_mode = LPM_MODE_SLEEP,
+	.standby_wake_sources = (lpm_standby_wake_source_t)0,
+#if BSP_FEATURE_LPM_HAS_SNOOZE
+	.snooze_cancel_sources = RA_NOT_DEFINED,
+	.snooze_request_source = RA_NOT_DEFINED,
+	.snooze_end_sources = (lpm_snooze_end_t)0,
+	.dtc_state_in_snooze = RA_NOT_DEFINED,
+#endif
+#if BSP_FEATURE_LPM_HAS_SBYCR_OPE
+	.output_port_enable = LPM_OUTPUT_PORT_ENABLE_RETAIN,
+#endif
+#if BSP_FEATURE_LPM_HAS_DEEP_STANDBY
+	.io_port_state = LPM_IO_PORT_NO_CHANGE,
+	.power_supply_state = LPM_POWER_SUPPLY_DEEP_STANDBY_MODE1,
+	.deep_standby_cancel_source = (lpm_deep_standby_cancel_source_t)0,
+	.deep_standby_cancel_edge = (lpm_deep_standby_cancel_edge_t)0,
+#endif
+#if BSP_FEATURE_LPM_HAS_PDRAMSCR
+	.ram_retention_cfg.ram_retention = (uint8_t)(0),
+	.ram_retention_cfg.tcm_retention = true,
+#endif
+#if BSP_FEATURE_LPM_HAS_DPSBYCR_SRKEEP
+	.ram_retention_cfg.standby_ram_retention = true,
+#endif
+#if BSP_FEATURE_LPM_HAS_LDO_CONTROL
+	.ldo_standby_cfg.pll1_ldo = false,
+	.ldo_standby_cfg.pll2_ldo = false,
+	.ldo_standby_cfg.hoco_ldo = false,
+#endif
+	.p_extend = NULL,
+};
+lpm_instance_ctrl_t g_lpm_sw_standby_ctrl;
+const lpm_cfg_t g_lpm_sw_standby_cfg = {
+	.low_power_mode = LPM_MODE_STANDBY,
+	.standby_wake_sources = LPM_STANDBY_WAKE_SOURCE_IRQ13 | LPM_STANDBY_WAKE_SOURCE_ULP0U |
```

Comment:
and here

Reply:
@nordicjm , Oops sorry I missed them, it's fixed. Thank you